runtime/doc/lsp.txt

```diff
@@ -425,6 +425,14 @@ therefore the linked highlights), use the |:highlight| command: >
     highlight LspDiagnosticsDefaultError guifg="BrightRed"
 <
 
+Note: Most custom colorschemes call `highlight clear` by convention on
+initialization. If the colorscheme does not define highlight groups for
+LSP diagnostics, you can set the highlights after the colorscheme has
+initialized with an autocommand. You will need to set this autocommand
+before enabling the colorscheme: >
+
+   autocmd ColorScheme * call v:lua.vim.lsp.diagnostic._define_default_signs_and_highlights()
```

Comment:
Does this function need to be promoted to public? Seems a bit weird to recommend using an internal function.